src/Controller/HomeController.php

```diff
@@ -2,17 +2,38 @@
 
 namespace App\Controller;
 
+use App\Repository\SnowFigureRepository;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
 
 class HomeController extends AbstractController
 {
     #[Route('/', name: 'home')]
-    public function index(): Response
+    public function index(SnowFigureRepository $snowFigureRepository): Response
     {
+        //list of published figures, by decreasing id
+        $figuresPublished = $snowFigureRepository->findBy(['publish' => '1'], ['id' => 'DESC']);
```

Comment:
Tu dois utiliser ton FigureManager pour faire ce traitement. 
```suggestion
        $nbPulishedFigures= $figureManager->countPublishedFigures();
```

Reply:
Voir [PR28](https://github.com/vanmarcke/ocr_projet6_SnowTricks/pull/69#discussion_r801070436)

Reply:
Done